docs/installation/manage/_index.md

```diff
@@ -10,97 +10,134 @@ images: ["/installation/img/thumbnails/manage.png"]
 
-{{< tabs name="Starting and stopping viam-server">}}
+## Run `viam-server`
+
+Select the tab for your platform:
+
+{{< tabs name="Managing viam-server">}}
 {{% tab name="Linux"%}}
 
-### As a system service
+### Run as a system service
 
-After setting up the system service per the [Linux install instructions](/installation/#install-viam-server), the AppImage binary will be located at <file>/usr/local/bin/viam-server</file>, and a systemd service file will be placed at <file>/etc/systemd/system/viam-server.service</file>.
+After [installation](/installation/#install-viam-server), the `viam-server` [AppImage](https://appimage.org/) binary will be located at <file>/usr/local/bin/viam-server</file>, and a `systemd` service file will be placed at <file>/etc/systemd/system/viam-server.service</file>.
+By default, `viam-server` is configured to start when the machine boots.
 
-By default, the `viam-server` is configured to start when the machine boots.
+Running `viam-server` as a system service is the recommended method for Linux.
 
-Sometimes you may want to manually start, stop or restart the `viam-server` systemd service, for instance, when troubleshooting.
-You can use the following commands to do so:
+You can use the following commands to manage `viam-server` when installed as a system service.
+These commands require that you store your configuration file at <file>/etc/viam.json</file>.
 
-Start:
+#### Start
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
 sudo systemctl start viam-server
 ```
 
-Stop:
+#### Stop
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
 sudo systemctl stop viam-server
 ```
 
-Restart:
+#### Restart
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
 sudo systemctl restart viam-server
 ```
 
-Enable (start automatically after boot):
+#### Enable (start automatically with system boot, default)
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
 sudo systemctl enable viam-server
 ```
 
-Disable (do not start automatically after boot):
+#### Disable (do not start automatically with system boot)
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
 sudo systemctl disable viam-server
 ```
 
 <br>
 
-### From the command line
+### Run from the command line
+
+When running `viam-server` on the command line, you can use the following commands to manage the process.
+If `viam-server` is already running as a system service, be sure to stop the service first before using these commands.
+
+#### Start
+
+Run the following on the command line to start `viam-server`, providing the path to your own configuration file:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+sudo viam-server -config /path/to/my/config.json
+```
+
+If you followed our [installation steps](/installation/#install-viam-server), your robot's configuration file is available at <file>/etc/viam.json</file>.

```

Comment:
```suggestion
If you followed these [installation steps](/installation/#install-viam-server), your robot's configuration file is available at <file>/etc/viam.json</file>.
```

the/these - to avoid the first person pronoun.